src/include/sof/audio/module_adapter/module/generic.h

```diff
@@ -212,6 +212,9 @@ struct processing_module {
 	 */
 	bool stream_copy_single_to_single;
 
+	/* flag to insure that module is loadable */
```

Comment:
this is really more, I would explain the high level differences here in an inline comment.